src/main/kotlin/fm/unit/UnitApp.kt

```diff
@@ -93,37 +121,45 @@ fun Application.module() {
         static("/static") {
             files("static")
         }
-        route("/reports") {
-            get("{key...}") {
-                // TODO(karsten): Lots of error handling and we don't support arbitrary prefix depth.
-                val key = call.parameters.getAll("key")?.joinToString("/") ?: ""
-
-                val reports = testReports()// TODO(karsten): fetch from database
-                val summary = ProjectSummary(reports)
-                call.respond(template(summary))
+        route("/reports/{organization}/{repository}/{prefix}") {
```

Comment:
`//TODO: we should follow github schema /{organisation}/{repository}/{prefix}/reports`